libraries/AP_Math/definitions.h

```diff
@@ -46,8 +33,8 @@
 // GPS Specific double precision conversions
 // The precision here does matter when using the wsg* functions for converting
 // between LLH and ECEF coordinates.
-#define DEG_TO_RAD_DOUBLE 0.0174532925199432954743716805978692718781530857086181640625
-#define RAD_TO_DEG_DOUBLE 57.29577951308232286464772187173366546630859375
+static const double DEG_TO_RAD_DOUBLE = 0.0174532925199432954743716805978692718781530857086181640625;
+static const double RAD_TO_DEG_DOUBLE = 57.29577951308232286464772187173366546630859375;
```

Comment:
these won't give the right values. If we use this:
static const double DEG_TO_RAD_DOUBLE = asin(1)/90;
static const double RAD_TO_DEG_DOUBLE = 1/DEG_TO_RAD_DOUBLE;
then we'll get the right values independent of compiler setting.
Otherwise we'd need a definitions.cpp and a special build rule to not force literal float for that file.